src/DevChatter.Bot.Core/Commands/TopCommand.cs

```diff
@@ -33,7 +33,8 @@ public static string GenerateMessage(List<ChatUser> topUsers)
 
         public List<ChatUser> GetTopUsers()
         {
-            return new List<ChatUser>(Repository.List<ChatUser>().Where(u => u.Role != UserRole.Streamer).OrderByDescending(b => b.Tokens).Take(5));
+            return new List<ChatUser>(Repository.List<ChatUser>().Where(u => u.Role != UserRole.Streamer)
+                .OrderByDescending(b => b.Tokens).Take(5));
```

Comment:
5 is a magic number. Consider putting it in a variable

Reply:
Agreed. I'll get these types of things